awscli/customizations/cloudformation/deploy.py

```diff
@@ -291,12 +315,12 @@ def _run_main(self, parsed_args, parsed_globals):
                            parameters, parsed_args.capabilities,
                            parsed_args.execute_changeset, parsed_args.role_arn,
                            parsed_args.notification_arns, s3_uploader,
-                           tags,
+                           tags, parsed_args.disable_rollback,
                            parsed_args.fail_on_empty_changeset)
 
     def deploy(self, deployer, stack_name, template_str,
                parameters, capabilities, execute_changeset, role_arn,
-               notification_arns, s3_uploader, tags,
+               notification_arns, s3_uploader, tags, disable_rollback=False,
```

Comment:
We likely want this as the last arg to guarantee backwards compatibility. Anyone customizing the CLI and passing in `fail_on_empty_changeset` as a positional arg may break with this change.

Reply:
fixed in 649a3b9 - made `disable_rollback=False` the final arg.